pithos/pithos.py

```diff
@@ -300,7 +300,7 @@ def init_core(self):
         self._query_position = Gst.Query.new_position(Gst.Format.TIME)
         self._query_buffer = Gst.Query.new_buffering(Gst.Format.PERCENT)
 
-        self.player = Gst.ElementFactory.make("playbin", "player")
+        self.player = Gst.ElementFactory.make("playbin3", "play")
```

Comment:
You can keep the name `player`, seems more clear.

Reply:
As in, _self.player_? Agreed. Also means a lot fewer code changes.

Reply:
The second argument is just a name, don't change it.